scripts/lint-linkcheck.mjs

```diff
@@ -173,6 +187,61 @@ class BrokenLinkChecker {
 		console.log();
 	}
 
+	outputSourceFileAnnotations (brokenLinks) {
+		const annotations = [];
+
+		// Collect all unique pathnames that had broken links
+		const pathnames = [...new Set(brokenLinks.map(brokenLink => brokenLink.page.pathname))];
```

Comment:
Nit, but I don't think you need to spread the Set?

```suggestion
		const pathnames = new Set(brokenLinks.map(brokenLink => brokenLink.page.pathname));
```

Reply:
Haha, good eye! Yeah, I wouldn't need to spread it in this case. It's just a personal preference to keep everything arrays in my code as I don't have to think twice if I need to access `.length` or `.size`, or if it's `.push` or `.add`.